simapp/app_v2.go

```diff
@@ -97,6 +97,9 @@ func init() {
 	DefaultNodeHome = filepath.Join(userHomeDir, ".simapp")
 }
 
+// TODO HV2: To fix many tests, we need to implement https://polygon.atlassian.net/browse/POS-2540
+//  and change NewSimApp function accordingly
```

Comment:
This version of the app uses depinject so I think we can ignore. We'd have to focus on the v1 version of the app.

Reply:
Yes but all the tests I had to skip when replacing `staking` with `stake` are failing because they leverage this function.